data/tools/wmlunits

```diff
@@ -584,10 +630,13 @@ if __name__ == '__main__':
     ap.add_argument("-a", "--addons",
                     help="Specify path to a folder with all addons. This should be " +
                     "outside the user config folder.")
-    ap.add_argument("-L", "--list",
-                    help="List available eras and campaigns.")
-    ap.add_argument("-B", "--batch",
-                    help="Batch process the given list.")
+    ap.add_argument("-L", "--list", nargs='*', # Distinguish 3 states: [--list [FILE]]
```

Comment:
For both `--list` and `--batch`, `nargs='?'` looks like a better solution than `nargs='*'`: it allows specifying both a default value to be used if the switch is missing and a const value to be used if the switch is used without arguments.

Reply:
``nargs='?'`` doesn't allow customizing the argument-less ``--batch`` to match ``--file``. In any case, @soliton- seems to strongly oppose the usage of variadic arguments.